src/main/java/org/elasticsearch/action/admin/cluster/snapshots/delete/TransportDeleteSnapshotAction.java

```diff
@@ -64,7 +64,7 @@ protected DeleteSnapshotResponse newResponse() {
 
     @Override
     protected ClusterBlockException checkBlock(DeleteSnapshotRequest request, ClusterState state) {
-        return state.blocks().indexBlockedException(ClusterBlockLevel.METADATA, "");
+        return state.blocks().indexBlockedException(ClusterBlockLevel.METADATA_WRITE, "");
```

Comment:
same as above, I guess this should be consistent with what we chose above. Do we even have to check for blocks here though?